demo/src/screens/PlaygroundScreen.tsx

```diff
@@ -1,8 +1,9 @@
 import React, {Component} from 'react';
-import {View, Text, Card, TextField, Button} from 'react-native-ui-lib'; //eslint-disable-line
+import {View, Text, Card, Avatar, TextField, Button} from 'react-native-ui-lib'; //eslint-disable-line
```

Comment:
Same here, you can remove the `//eslint-disable-line`

Reply:
Fixed